### ISSUE TITLE
Add additional probability distributions for estimates (e.g. student-t, metalog)

### ISSUE THREAD

**Issue by thomaslow** [CWWhitney/decision_ui#11]
The [decisionSupport](https://cran.r-project.org/web/packages/decisionSupport/index.html) CRAN package supports more probability distributions than currently implemented in the user interface, e.g.:
- `beta`
- `cauchy`
- `logis` - Logistic
- `t` - student-t
- and many more, see [documentation](https://cran.r-project.org/web/packages/decisionSupport/refman/decisionSupport.html#random.estimate1d)

The main problem is that distribution parameters need to be found (like mean and variance of a normal distribution) based on confidence interval lower and upper bounds. For many distributions, this is not possible via a closed-form expression. Instead, parameters are estimated via a fitting algorithm, see [rdistq_fit](https://cran.r-project.org/web/packages/decisionSupport/refman/decisionSupport.html#topic+rdistq_fit) based on the R package [rriskDistributions::rriskFitdist.perc](https://cran.r-project.org/web/packages/rriskDistributions/refman/rriskDistributions.html#rriskFitdist.perc). Internally, it uses the [`stats::optim`](https://stat.ethz.ch/R-manual/R-devel/library/stats/html/optim.html) optimizer with the "L-BFGS-B" method. This is a variant of [Newton's method](https://en.wikipedia.org/wiki/Newton%27s_method_in_optimization).

As far as I know, there is no equivalent implementation for the `rriskDistributions` package in Javascript. In theory, it should be possible to achieve similar results with other optimization methods, e.g. gradient descent as provided by TensorflowJS. However, TensorflowJS doesn't even provide PDF and CDF of common probability distributions, so other implementations need to be found as well, e.g. [jstat](https://github.com/jstat/jstat). In summary, adding new distributions will involve a lot work.

An exception could be `unif` ([uniform distribution](https://en.wikipedia.org/wiki/Continuous_uniform_distribution)), which doesn't require parameter fitting.

Possible other distributions that are not implemented in the decisionSupport CRAN package:
- [Metalog](http://metalogdistributions.com/)
